app/src/js/components/Header/header.js

```diff
@@ -15,7 +15,9 @@ const paths = [
   ['Users', '/users', 'USER'],
   ['Groups', '/groups', 'GROUP'],
   ['Roles', '/roles', 'ROLE'],
-  ['Conversations', '/conversations', 'CONVERSATION']
+  ['Conversations', '/conversations', 'CONVERSATION'],
+  ['Inputs', '/inputs', 'INPUTS'],
```

Comment:
We probably don't want to start adding headers for these admin only endpoints otherwise, we'll have to refactor the header as it starts to break if you add too many.

Reply:
Sure, I made the changes and pushed.